src/terminal/input/terminalInput.cpp

```diff
@@ -530,7 +530,7 @@ bool TerminalInput::HandleKey(const IInputEvent* const pInEvent)
 
     // This section is similar to the Alt modifier section above,
     // but handles cases without Ctrl modifiers.
-    if (keyEvent.IsAltPressed() && keyEvent.GetCharData() != 0)
+    if (keyEvent.IsAltPressed() && !keyEvent.IsCtrlPressed() && keyEvent.GetCharData() != 0)

```

Comment:
I've had to add these two additional checks, because `Alt+Ctrl` combinations actually sometimes do carry character data!
For instance `Ctrl+Alt+<` will send an `KeyEvent` with the CharData being `|`. 😲
The old condition here (`keyEvent.IsAltPressed() && keyEvent.GetCharData() != 0`) would thus incorrectly match and `Ctrl+Alt+<` would send `^[|` instead of `|`.